components/news_summary_process.py

```
@@ -28,45 +43,67 @@ def run():
+            st.session_state['summary_len'] = 'long'
     else:
         lang = "unknown"
 
-    if 0 < len(text) < 100:
-        st.warning("입력 길이가 너무 짧습니다 (권장 길이: 100~1000자)")
 
-    length_option = st.sidebar.selectbox(
-        "요약문 길이 선택",
-        ("짧게 (약 100~300자)", "중간 (약 200~400자)", "길게 (약 400~600자)"),
-    )
+def show_summary_title(text, lang):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)